### ISSUE TITLE
Add Accessibility Statement to dataverse.no

### ISSUE THREAD

**Issue by philippconzett** [DataverseNO/local.dataverse.no#34]
A link to an Accessibility Statement  should be added to dataverse.no. This is a recent national requirement in Norway, as of February 1, 2023.

I suggest to add this in the footer, to the right of "Contact". Thus add the following after "Contact":

` | <a href="https://uustatus.no/nn/erklaringer/publisert/8374ca6b-85e3-4b4c-9d8e-56d37d9b0088" target="_blank">Tilgjengelegheitserklæring (Accessibility Statement)</a>
`

This should also be reflected in the new, cloud-based instance of DataverseNO.
Thanks!

**Comment by oodu** [comment]
Accessibility Statement is added to local dataverse.no and local test.dataverse at the footer:(/usr/local/payara5/glassfish/domains/domain1/applications/dataverse-5.6/dataverse_footer.xhtml
datapwd

**Comment by philippconzett** [comment]
Takk!

**Comment by Louis-wr** [comment]
@oodu could you also update the git as we disccused?
https://github.com/DataverseNO/dataverse-docker/blob/dataverse.no/distros/dataverse.no/modification/custom-footer.html

**Comment by Louis-wr** [comment]
@oodu I thought you added this to the footer?
![image](https://github.com/DataverseNO/local.dataverse.no/assets/85620187/1ae419e7-14a8-4bb6-86a9-19497a104517)
I don't see it in the file history, did you forget to push?
